### DISCUSSION TITLE
VirtualMin + Outline

### DISCUSSION THREAD

**Discussion by josidu** [outline/outline#5435] [Self Hosting]
Am I overthinking or can I install outline all by my self on my VirtualMin VPS?
I'd create a outline.mydomain.com account and then access console and follow the guide "[Install from Source](https://docs.getoutline.com/s/hosting/doc/from-source-BlBxrNzMIP)"
Is that right?! Is that gonna work?

**Comment by tommoor** [comment]
I don’t see why not
